client/components/jetpack/intro-pricing-banner/index.tsx

```diff
@@ -16,9 +16,10 @@ const Banner: React.FC = () => {
 				</p>
 				<p className="intro-pricing-banner__call-to-action">
 					{ translate(
-						'Get the perfect Jetpack for your site with %(percent)d%% off the first term. *Try it risk free with our %(days)d-day money-back guarantee.',
+						'Get the perfect Jetpack for your site with %(percent)d%% off the first term. Try it risk free with our %(days)d-day money-back guarantee.*',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.